File: C/length_of_last_word.c
int lengthOfLastWord(char * s){
    int length = 0;
    bool flag = false;
        
    for(int i = strlen(s)-1; i >= 0; i--){
        if(s[i] != ' '){
            length ++;
            flag = true;
            continue;
        }   
        else{
            if(flag) break;
        }
    }
    return length;
}

File: Python3/length_of_last_word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        length = 0
        flag = False
        
        for i in range(len(s)):
            if s[-1-i] != ' ': 
                length += 1
                flag = True
                continue
            else:
                if flag: break
                      
        return length
